.github/tests/production-example/pre-install.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+kubectl create namespace spire-system
+kubectl label namespace spire-system pod-security.kubernetes.io/enforce=privileged
+kubectl create namespace spire-server
+kubectl label namespace spire-server pod-security.kubernetes.io/enforce=restricted
```

Comment:
Since you are doing an install.sh, this stuff could just be moved there.

Reply:
Prefer to keep it here, in case future test setup refactors, this makes it very clear.